vcd/resource_vcd_vapp_vm.go

```diff
@@ -1086,3 +1177,24 @@ func readNetworks(vm govcd.VM, vapp govcd.VApp) ([]map[string]interface{}, error
 	}
 	return nets, nil
 }
+
+// isForcedConfiguration checks "customization" block in resource and checks if the value of field "force"
+// is set to "true". It returns false if the value is not set or is set to false
+func isForcedConfiguration(customizationBlock interface{}) bool {
```

Comment:
func name is misleading I would say, block custimization but func name is configuration

Reply:
renamed to `isForcedCustomization()`